wazo_agentd_client/commands/status.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from wazo_lib_rest_client.command import RESTCommand
+from .agents import _ResponseProcessor
```

Comment:
`_ResponseProcessor` should be renamed `ResponseProcessor` and moved to another module. Maybe to `wazo_agentd_client.client` or `wazo_agentd_client.helpers`.

Reply:
Done.